src/main/java/org/jabref/logic/net/URLDownload.java

```diff
@@ -305,11 +305,12 @@ public Path toTemporaryFile() throws IOException {
 
         // Take everything after the last '/' as name + extension
         String fileNameWithExtension = sourcePath.substring(sourcePath.lastIndexOf('/') + 1);
-        String fileName = FileUtil.getBaseName(fileNameWithExtension);
+        String fileName = "jabref-" + FileUtil.getBaseName(fileNameWithExtension);
```

Comment:
I had to think long about this. Since its a temporary file anyway, the filename doesn't matter. Thus, good to go!